packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -180,112 +366,99 @@ export function TelemetryView(): React.ReactElement {
 		}),
 	];
 
-	const log_view =
-		telemetryEvents !== undefined ? (
-			<>
-				<h3>Telemetry events (newest first):</h3>
-				<DataGrid
-					items={items}
-					columns={columns}
-					resizableColumns
-					columnSizingOptions={{
-						category: {
-							minWidth: 65,
-							idealWidth: 80,
-						},
-						eventType: {
-							minWidth: 60,
-							idealWidth: 150,
-						},
-						information: {
-							minWidth: 60,
-							idealWidth: 250,
-						},
-					}}
-				>
-					<DataGridHeader>
-						<DataGridRow style={{ whiteSpace: "normal" }}>
-							{({ renderHeaderCell }): JSX.Element => (
-								<DataGridHeaderCell>{renderHeaderCell()}</DataGridHeaderCell>
-							)}
-						</DataGridRow>
-					</DataGridHeader>
-					<DataGridBody<Item>>
-						{({ item, rowId }): JSX.Element => (
-							<DataGridRow<Item> key={rowId}>
-								{({ renderCell }): JSX.Element => (
-									<DataGridCell>{renderCell(item)}</DataGridCell>
-								)}
-							</DataGridRow>
-						)}
-					</DataGridBody>
-				</DataGrid>
-			</>
-		) : (
-			<Waiting label={"Waiting for Telemetry events"} />
-		);
+	/**
+	 * Gets list of valid categories for displayed telemetry events.
+	 * @returns list of option
+	 */
+	function getCategories(): { key: string; text: string }[] {
```

Comment:
Nit: I would recommend moving this to be above the rendering code that calls it.